src/pages/sponsors.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import Link from 'gatsby-link'
+import get from 'lodash/get'
+import Helmet from 'react-helmet'
+
+import logoElysian from '../assets/images/elysian-logo.png'
+
+class Generic extends React.Component {
+  render() {
+
+    return (
+      <div>
+        <Helmet title={get(this, 'props.data.site.siteMetadata.title')} />
+
+        <header id="header">
+            <h1>Sponsors</h1>
+            <p>the teams who help us support the community</p>
+        </header>
+
+        <div id="main">
+          <section id="content" className="main">
+            <ul className="features">
+              <li>
+                <h3><a href='https://elysiancoffee.com/'>Elysian Coffee Roasters</a></h3>
+                <p>
+                  Elysian Coffee Roasters have hosted our smaller community meetups at their
+                  café.
+                  Their crew is welcoming and their coffee is delicious ☕️
+                </p>
+              </li>
+              <li>
+                <h3><a href='https://7gate.vc'>7 Gate Ventures</a></h3>
+                <p>
+                  7 Gate Ventures have shared their spaces with several of our meetups.
+                  They focus on technology and the community powering it 💻
+                </p>
+              </li>
+            </ul>
+          </section>
```

Comment:
on a related note we totally could make a `sponsors.json` file or something (or if you wanted to go overkill, one of the gatsby data sources).  It could store an array of sponsors with description, url, and name, and they would just get rendered here in random order.

THIS IS PROBABLY OVERKILL FOR A SPONSOR PAGE OF 2, DON'T TAKE THIS SERIOUSLY

Reply:
Why even _write_ code if you're not going to over-engineer it? :trollface:

Reply:
this is true.  Also, within the review process, we've updated the number of sponsors to _4_

Reply:
Did I mention how much fun I have working with you both on this?! 💥🖤💥

Reply:
To get this merged, would it make sense to make a new issue for this?

Is it a candidate for a good-first-issue?

Reply:
Yeah, definitely cool landing without this change. Might be a good candidate for a NodeSchool Open Source Day PR. ;)